File: 백준/Bronze/5073. 삼각형과 세 변/삼각형과 세 변.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf-8').toString().trim().split('\n').map(x => x.split(' ').map(Number));
let answer = '';

for (let cs of input) {
    const [a, b, c] = cs.sort((a, b) => a - b);
    let result;
    
    if (a * b * c === 0) break;
    
    if (a + b <= c) {
        result = 'Invalid';
    } else if (a === b && b === c) {
        result = 'Equilateral';
    } else if (a === b || b === c || c === a) {
        result = 'Isosceles';
    } else {
        result = 'Scalene';
    }
    
    answer += `${result}\n`;
}

console.log(answer);

File: 백준/Bronze/5073. 삼각형과 세 변/삼각형과 세 변.py
import sys

for row in sys.stdin.readlines():
    x, y, z = sorted(map(int, row.split()))
    if x + y + z == 0:
        break
        
    if x + y <= z:
        print("Invalid")
    elif x == y and y == z and z == x:
        print("Equilateral")
    elif x != y and y != z and z != x:
        print("Scalene")
    else:
        print("Isosceles")